packages/terra-form-input/src/Input.scss

```diff
@@ -0,0 +1,92 @@
+:local {
+  .form-input {
+    background-color: var(--terra-form-input-background-color, #fff);
+    background-image: var(--terra-form-input-focus-animation-background, none);
+    background-position: 0 100%;
+    background-repeat: no-repeat;
+    background-size: var(--terra-form-input-focus-background-start, 0 0);
+    border: var(--terra-form-input-border, 1px solid #4a4a4a);
+    border-radius: var(--terra-form-input-border-radius, 0.2rem);
+    box-shadow: var(--terra-form-input-box-shadow, none);
+    color: var(--terra-form-input-color, #000);
+    font-family: inherit;
+    font-size: var(--terra-form-input-font-size, 1.143rem);
+    line-height: var(--terra-form-input-line-height, 1.143);
+    opacity: var(--terra-form-input-opacity, 1);
+    padding-bottom: var(--terra-form-input-padding-bottom, 0.571rem);
+    padding-left: var(--terra-form-input-padding-left, 0.357rem);
+    padding-right: var(--terra-form-input-padding-right, 0.357rem);
+    padding-top: var(--terra-form-input-padding-top, 0.571rem);
+    transition-duration: var(--terra-form-input-transition-duration, 0s);
+    transition-property: background-size;
+    transition-timing-function: var(--terra-form-input-transition-timing-function, ease);
+    width: 100%;
+
+    &:hover {
+      border: var(--terra-form-input-hover-border, 1px solid #4a4a4a);
+    }
+
+    &[required] {
+      box-shadow: unset;
+    }
+
+    &[disabled] {
+      background-color: var(--terra-form-input-disabled-background-color, rgba(0, 0, 0, 0.05));
```

Comment:
Need to check with @neilpfeiffer if we need styling of background, border, and color in the disabled state or if we can just adjust the opacity.